pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2404,12 +2443,16 @@
     %~2020.5.29  ..^$(ax old)
   ::
       %~2019.10.6
+    =^  success  bindings.server-state.old
+      %+  insert-binding
+        [[~ /~/logout] ~ [%logout ~]]
```

Comment:
@eglaysher not entirely sure how bad it is to use the `~` duct here, but didn't want to just make one up either. In practice only Eyre itself can undo these bindings anyway, so this shouldn't be the end of the world, right?

Reply:
That's probably correct, but having null ducts around seems like the sort of thing which eventually causes chaos...

Reply:
A null duct is definitely risky and weird.  If you ever pass on that duct and then someone gives a response, that'll go straight to unix instead of eyre, giving strange errors.  If you give on that duct... undefined behavior?  Worst of all, if someone stumbles on an empty duct in the wild, you can't exactly grep for `~`.

I'd hand-create a greppable duct for this purpose.  Something like ~[/logout] should be fine.  Still don't give on that duct or it'll go to unix, but at least if you pass on it it'll come back correctly, and it's greppable.

Reply:
The only place the binding ducts are used is in verifying binding "ownership", so that only the original creator of it can delete it. This seems.. potentially fragile, but that's a discussion for another time. I'll put a fake duct there for now, thanks all!